packages/adapter-node/src/server.js

```diff
@@ -1,57 +1,13 @@
-import compression from 'compression';
-import fs from 'fs';
-import polka from 'polka';
-import { dirname, join } from 'path';
-import sirv from 'sirv';
-import { URL, fileURLToPath } from 'url';
-// eslint-disable-next-line import/no-unresolved
-import { get_body } from '@sveltejs/kit/http';
-// App is a dynamic file built from the application layer.
-/*eslint import/no-unresolved: [2, { ignore: ['\.\/app\.js$'] }]*/
-import * as app from './app.js';
-
-const __dirname = dirname(fileURLToPath(import.meta.url));
+import { createServer } from './server_base.js';
```

Comment:
what if we kept the `createServer` logic in `src/server.js` and put the startup logic in `files/index.js`? That would match the output filename (we write to `index.js` so that you can do `node build` to start the built app), and it shouldn't break anything